test/data_management/test_dataset_building.py

```diff
@@ -1,166 +1,109 @@
 # coding: utf-8
 
-import os
+from pathlib import Path
 import shutil
 import pytest
 from AxonDeepSeg.data_management.dataset_building import *
 
 
 class TestCore(object):
     def setup(self):
-        self.fullPath = os.path.dirname(os.path.abspath(__file__))
+        # Get the directory where this current file is saved
+        self.fullPath = Path(__file__).resolve().parent
         # Move up to the test directory, "test/"
-        self.testPath = os.path.split(self.fullPath)[0]
+        self.testPath = self.fullPath.parent
 
-        self.rawPath = os.path.join(
-            self.testPath,
-            '__test_files__',
-            '__test_patch_files__',
-            'raw'
+        _create_new_test_folder = (
+            lambda s: self.testPath
```

Comment:
Is lambda syntax really necessary to concate multiple Paths with `/` like we do with join?

Reply:
Oh I see what you did now